spec.html

```diff
@@ -99,19 +98,21 @@ <h1>_NativeError_ Object Structure</h1>
 
     <emu-clause id="sec-properties-of-nativeerror-instances">
       <h1>Properties of _NativeError_ Instances</h1>
-      <p>_NativeError_ instances are ordinary objects that inherit properties from their _NativeError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error, _NativeError_, or _AggregateError_ instances.</p>
+      <p>_NativeError_ instances are ordinary objects that inherit properties from their _NativeError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error<ins>, AggregateError</ins> or _NativeError_ instances.</p>
```

Comment:
`<ins>` should be placed after comma

Reply:
The comma is not present in <https://tc39.es/ecma262/#sec-properties-of-nativeerror-instances>.